ui/src/org.ts

```diff
@@ -0,0 +1,76 @@
+import * as api from "./api"
+import { MessageType, Transaction } from './transaction';
+
+
+export enum RegistrationFlowState {
+  NameSelection,
+  TransactionConfirmation
+}
+
+const imageUrl =
+  "https://pbs.twimg.com/profile_images/378800000411356732/e8b1b7f0bd07d4d948cb2da25e221673_400x400.jpeg";
+
+
+interface Org {
+  name: string;
+  avatar: {
+    imageUrl: string;
+  };
+}
+
+// Name validation
+const VALID_NAME_MATCH = new RegExp("^[a-z0-9][a-z0-9_-]+$");
+export const nameConstraints = {
+  presence: {
+    message: `Org name is required`,
+    allowEmpty: false
+  },
+  format: {
+    pattern: VALID_NAME_MATCH,
+    message: `Org name should match [a-z0-9][a-z0-9_-]+`
+  }
+};
+
+// DUMMY DATA
+
+export const org: Org = {
+  name: "",
+  avatar: {
+    imageUrl: imageUrl
```

Comment:
Same as above, org doesn't have a image url for now. It only has the avatarFallback.

Reply:
TBD in subsequent api integration pr